### PR TITLE
feat: various stylistic fixes and improvements

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Attempting to approximate @lanceplaine's figma design more closely.

### CODE REVIEW

**Comment by lanceplaine** [comment]
- [ ] Is there a way not to have the scrolling bar red? And keep the regular color. I find it distracting. Also it shouldn't always be show like this. It should be the browser decision. On Safari for example it's hidden unless the user is scroll 

Scrollbar

**Comment by lanceplaine** [comment]
- [ ] The header and cards should be aligned
Header and cards

**Comment by lanceplaine** [comment]
- [ ] Icons are missing.

You can either use FontAwesome to do it or use assets. 
I attached here the PNG exports.

[Resource icons.zip](https://github.com/ethereum-optimism/community-hub/files/7825448/Resource.icons.zip)

**Comment by lanceplaine** [comment]
- [ ] When hovering only change the shadow intensity, not the size of the text and card

**Default state:**
```
box-shadow: 0px 8px 16px -6px rgba(20, 23, 26, 0.08);
box-shadow: 0px 6px 8px -6px rgba(20, 23, 26, 0.12);
```

**Hover state:**
```
box-shadow: 0px 12px 42px -4px rgba(20, 23, 26, 0.12);
box-shadow: 0px 8px 18px -6px rgba(20, 23, 26, 0.12);
```

CleanShot 2022-01-06 at 16 24 00@2x

CleanShot 2022-01-06 at 16 25 48@2x

**Comment by lanceplaine** [comment]
- [ ] Update the font size, weight, color and family for the title and subtitle in the card.
CleanShot 2022-01-06 at 16 28 55@2x
CleanShot 2022-01-06 at 16 28 48@2x

**Comment by lanceplaine** [comment]
- [ ] Some pages are a 404 but I'm guessing it will be solve in other PRs

**Comment by lanceplaine** [comment]
Looks good to me otherwise 👍

**Comment by smartcontracts** [comment]
Only way to get the homepage to look right was to eject the theme and modify it. Might mean some maintenance overhead down the line. Unfortunately Vuepress 1.x does not allow us to easily extend the theme for ourselves without ejecting it like I did in this PR.

**Comment by lanceplaine** [comment]
- [ ] For the icons you are using the `solid` style instead of `regular`. Apart from this everything else looks great! 🥳

Looks good to me

**Comment by elenadimitrova** [comment]
I also really want to have the Community dropdown the way Kevin designed it and the same content, i.e. links with logos to Discord, Github etc
Screenshot 2022-01-07 at 11 58 40

**Comment by smartcontracts** [comment]
> I also really want to have the Community dropdown the way Kevin designed it and the same content, i.e. links with logos to Discord, Github etc

Just pushed an update for this

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -43,8 +46,51 @@ module.exports = {
...
             text: 'Discord',
             link: 'https://discord.optimism.io',
+          },
+          {
+            icon: 'github',
+            iconPrefix: 'fab fa-',
+            iconClass: 'color-github',
+            text: 'GitHub',
+            link: 'https://github.com/ethereum-optimism/optimism',
+          },
```
Comment (elenadimitrova) [inline src/.vuepress/config.js:61]: Can we add the logos to these links? Like in @lanceplaine 's design

**Comment by lanceplaine** [comment]
Looks good to me

**Pull request merged**